src/slashers/VetoableSlasher.sol

```diff
@@ -0,0 +1,109 @@
+// SPDX-License-Identifier: BUSL-1.1
+pragma solidity ^0.8.12;
+
+import {IStrategy} from "eigenlayer-contracts/src/contracts/interfaces/IStrategy.sol";
+import {SlasherBase} from "./SlasherBase.sol";
+
+contract VetoableSlashing is SlasherBase {
+    struct SlashingRequest {
+        address operator;
```

Comment:
abstract a SlashingParams in here

Reply:
Good call, swapped it out int the function params too [337d561](https://github.com/Layr-Labs/eigenlayer-middleware/pull/310/commits/337d561b1da9123362adfe56b0b638595b54260c)